### ISSUE TITLE
fat gem does not include binary for Ruby 2.3 

### ISSUE THREAD

**Issue by cosmo0920** [nurse/strptime#9]
I got following error messages in AppVeyor:

```log
C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/strptime-0.1.7-x64-mingw32/lib/strptime.rb:5:in `require': cannot load such file -- strptime/strptime (LoadError)
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/strptime-0.1.7-x64-mingw32/lib/strptime.rb:5:in `rescue in <top (required)>'
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/strptime-0.1.7-x64-mingw32/lib/strptime.rb:2:in `<top (required)>'
	from C:/projects/fluentd/lib/fluent/parser.rb:20:in `require' 
	from C:/projects/fluentd/lib/fluent/parser.rb:20:in `<top (required)>'
	from C:/projects/fluentd/lib/fluent/test/parser_test.rb:17:in `require' 
	from C:/projects/fluentd/lib/fluent/test/parser_test.rb:17:in `<top (required)>'
	from C:/projects/fluentd/lib/fluent/test.rb:23:in `require' 
	from C:/projects/fluentd/lib/fluent/test.rb:23:in `<top (required)>'
	from C:/projects/fluentd/test/helper.rb:44:in `require' 
	from C:/projects/fluentd/test/helper.rb:44:in `<top (required)>'
	from C:/projects/fluentd/test/config/test_config_parser.rb:1:in `require_relative'
	from C:/projects/fluentd/test/config/test_config_parser.rb:1:in `<top (required)>'
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:15:in `require'
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:15:in `block in <main>'
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:4:in `select'
	from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/rake-11.1.2/lib/rake/rake_test_loader.rb:4:in `<main>'
rake aborted!
```

ref: https://ci.appveyor.com/project/cosmo0920/fluentd/build/71/job/ar8l5ot24u74qqfh#L73

Could you provide fat gems which include binaries for Ruby 2.3?

**Comment by nurse** [comment]
Released v0.1.8 and deployed gems including 2.3.0, thanks!

**Comment by cosmo0920** [comment]
Awesome!
Thank you for fixing this!
